docs/cryoet_data_portal_docsite_quick_start.md

```diff
@@ -17,15 +27,15 @@ python -m venv ./venv
 source ./venv/bin/activate
 ```
 
-Install the latest `cryoet-data-portal` package via pip:
+Install the latest `cryoet_data_portal` package via pip:
 
 ```shell
-pip install -U cryoet-data-portal
+pip install cryoet_data_portal
```

Comment:
I believe this got missed. Can we have this be the same as before `pip install -U cryoet-data-portal` ?

Reply:
I switched it back.